src/mirakuru/base.py

```diff
@@ -226,7 +227,14 @@ def _kill_all_kids(self, sig):
         pids = processes_with_env(ENV_UUID, self._uuid)
         for pid in pids:
             log.debug("Killing process %d ...", pid)
-            os.kill(pid, sig)
+            try:
+                os.kill(pid, sig)
+            except OSError as err:
+                if err.errno != errno.ESRCH:
```

Comment:
if the ESRCH is `no such process` don't we want this to be `err.errno == errno.ESRCH` 
https://docs.python.org/2/library/errno.html ?

Reply:
You're absolutely right. Damn copy-paste :confused:

Reply:
Oh, that's why our builds started getting flaky again! :unamused: